### PR TITLE
Various mappings in data/com/mojang/realmsclient

### PR DESCRIPTION
**Pull request by PiTheGuy**

### CODE REVIEW

File: data/com/mojang/realmsclient/client/FileDownload.mapping [hunk 1]
```diff
@@ -1,4 +1,28 @@
 CLASS com/mojang/realmsclient/client/FileDownload
+	METHOD download (Lcom/mojang/realmsclient/dto/WorldDownload;Ljava/lang/String;Lcom/mojang/realmsclient/gui/screens/RealmsDownloadLatestWorldScreen$DownloadStatus;Lnet/minecraft/world/level/storage/LevelStorageSource;)V
+		ARG 2 worldName
```
Comment (Lanse505) [inline data/com/mojang/realmsclient/client/FileDownload.mapping:4]: Order of arguments are: ``download, worldName, status, source``

File: data/com/mojang/realmsclient/gui/screens/RealmsPlayerScreen.mapping [hunk 2]
```diff
@@ -23,3 +23,5 @@ CLASS com/mojang/realmsclient/gui/screens/RealmsPlayerScreen
 	CLASS InvitedObjectSelectionList
 		METHOD addEntry (Lcom/mojang/realmsclient/dto/PlayerInfo;)V
 			ARG 1 playerInfo
+		METHOD selectInviteListItem (I)V
+			ARG 1 player
```
Comment (Lanse505) [inline data/com/mojang/realmsclient/gui/screens/RealmsPlayerScreen.mapping]: this should be ``index`` to match super call.

**Review by Lanse505 — CHANGES_REQUESTED** [review]

**Comment by PiTheGuy** [comment]
Suggestions have been applied

**Review by Lanse505 — APPROVED** [review]
lgtm

**Pull request merged**